### DISCUSSION TITLE
Is it possible to place a custom component on the XAxis with this library?

### DISCUSSION THREAD

**Discussion by wacci6** [Abhinandan-Kushwaha/react-native-gifted-charts#902] [Q&A] [answered]
I’m creating an app to manage the clothes I own, and I want to make it visually easy to see how many clothes I have in each color. I’ve attached a simple UI design image.

<img width="274" alt="スクリーンショット 2024-11-09 18 30 41" src="https://github.com/user-attachments/assets/5f221f2c-60d0-430c-b7a7-b8ebcad9f55f">

Currently, it seems like only text can be displayed, but is there any way to achieve this?
If not, I’d like to request a feature addition.

**Comment by Abhinandan-Kushwaha** [comment, marked as answer]
Yes @wacci6 You can use the property `labelComponent` in each item of the data array.
See https://github.com/Abhinandan-Kushwaha/react-native-gifted-charts/blob/master/docs/BarChart/BarChartProps.md#item-description-bardataitem
